scripts/import/import_clinvar_xml

```diff
@@ -1046,19 +1050,72 @@ sub get_variant{
       foreach my $vf (@$var_feats){
         warn "vf:". $vf->name . " allele string: ". $vf->allele_string . "\n";
       }
+    } else {
+      warn "no variation feature found via search by location identifier: $location_str\n" if $DEBUG ==1;
     }
-    else{
-      ## try to match to multi-allelic variants by location
-      return get_variant_by_slice_allele($record, $alt_allele);
-    }
-  }
-  else {
-    warn "No ref or alt allele or location found\n" if $DEBUG == 1;
-    return undef;
+  } else {
+    warn "no ref or alt or chr or start, skipping lookup by location identifier\n";
   }
+
+  #try to match by SPDI: longest allele format
+  #returns undef for ref==alt SPDIs
+  my ($var, $var_feats) = get_variant_via_SPDI ($record);
```

Comment:
Can we return the alt allele here too? (For e!105 if not e!104)

Reply:
The method does not change the alt allele. The alt stays the same as before from hgvs.

Reply:
If the variant is a deletion, HGVS reports only 'del' so we don't have the actual allele. We can obtain this from SPDI (with care)